server/src/com/cloud/network/vpc/VpcManagerImpl.java

```diff
@@ -501,7 +517,7 @@ private void validateConnectivtyServiceCapabilities(final Set<Provider> provider
             final Iterator iter = serviceCapabilityCollection.iterator();
 
             while (iter.hasNext()) {
-                final HashMap<String, String> svcCapabilityMap = (HashMap<String, String>) iter.next();
+                final HashMap<String, String> svcCapabilityMap = (HashMap<String, String>)iter.next();
```

Comment:
Please type to the ``Map`` interface rather than ``HashMap`` to reduce coupling.